### ISSUE TITLE
mockads mapping not getting applied properly. fixing _timestamp.

### ISSUE THREAD

**Issue by Analect** [DigElements/demo-app#1]
@flyntde 
Just posting here in case someone else is affected.
Running this against ES v. 2.3.3
Getting this error when pushing data to ES, since I think that way of enabling timestamp was [deprecated](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-timestamp-field.html) from version 2.
```
removing mockads index ...
creating mockads index ...
{"error":{"root_cause":[{"type":"mapper_parsing_exception","reason":"Mapping definition for [_timestamp] has unsupported parameters:  [store : true]"}],"type":"mapper_parsing_exception","reason":"Failed to parse mapping [_default_]: Mapping definition for [_timestamp] has unsupported parameters:  [store : true]","caused_by":{"type":"mapper_parsing_exception","reason":"Mapping definition for [_timestamp] has unsupported parameters:  [store : true]"}},"status":400}
importing mock ads data set into elasticsearch ...
```
As a result, city_agg was somehow ignoring the "not_analyzed" on city field and counting the individual terms "New", "York" and "City" instead of "New York City".

I changed the mapping in the bash file to this (only changed top part shown), which appeared to resolve the problem.

```
curl -XPUT clienta.analect.com:9200/mockads -d '
{
  "mappings" :{
    "ad" : {
      "_timestamp": {
	"enabled": true
	},
      "properties" : {
        "age" : {
          "type" : "long"
        },
        "city" : {
          "type" : "string",
          "index": "not_analyzed"
        },
```
Thanks for your work on this. Still trying to get my head around all the additional elements you have created and how they hang together!

**Comment by flyntde** [comment]
Thank you @Analect; I also discovered this today.  I am off this project now, and so I am no longer the owner of DigElements organization.  I created a new https://github.com/ElasticElements org where I am currently focused on implementing a few features such as filters and checklists for aggregation + filters.  The version for everything is at 0.0.x, meaning this is not ready for a full-featured application.  I will update all of the version numbers to 1.x.x when the suite of elements are ready for use.
